zcash_primitives/src/transaction/builder.rs

```diff
@@ -213,6 +213,7 @@ impl TransparentInputs {
     #[cfg(feature = "transparent-inputs")]
     fn apply_signatures(&self, mtx: &mut TransactionData, consensus_branch_id: u32) {
         for (i, info) in self.inputs.iter().enumerate() {
+            let mut sighash = [0u8; 32];
```

Comment:
Ooh, good catch! This wasn't caught because CI isn't enabling the feature flag. I'll have a think about how to handle that better in CI.

Please move this up by one line out of the loop.

Reply:
I'm trying to use `zcash/librustzcash` directly in Zecwallet Lite, removing the dependency on my custom fork. There's still a few things missing that I'm using in Zecwallet Lite, I'll make PRs for them.